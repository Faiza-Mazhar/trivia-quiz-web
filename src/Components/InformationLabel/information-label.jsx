import React from "react";
import "./information-label.style.scss";

const InformationLabel = ({ information }) => (
  <div className="label-container">
    <label className="label">{information}</label>
  </div>
);

export default InformationLabel;
