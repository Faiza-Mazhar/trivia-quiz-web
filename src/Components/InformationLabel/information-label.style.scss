.label-container {
  display: flex;
  flex-wrap: wrap;
  padding: 30px;

  .label {
    flex: 45%;
    width: auto;
    height: auto;
    color: white;

    letter-spacing: 0.5px;
    font-family: inherit;

    font-weight: bold;
    font-size: large;

    background-color: #556342;

    border: 1px solid #e2eee2;
    padding: 10px;
    border-radius: 5px;
    text-align: center;
  }
}
