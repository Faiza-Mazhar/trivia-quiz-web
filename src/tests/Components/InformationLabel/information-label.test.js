import InformationLabel from "../../../Components/InformationLabel/information-label";
import renderer from "react-test-renderer";

test("Information label is rendered is correct props", () => {
  const component = renderer.create(
    <InformationLabel information={"This is my information"} />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
