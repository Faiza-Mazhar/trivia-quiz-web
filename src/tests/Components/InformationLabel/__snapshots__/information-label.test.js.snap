// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information label renders correct props 1`] = `
<div
  className="label-container"
>
  <label
    className="label"
  >
    This is my information
  </label>
</div>
`;
